db/migrate/20230223110925_create_users.rb

```diff
@@ -0,0 +1,12 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.string :photo
+      t.text :bio
+      t.integer :PostsCounter
```

Comment:
- [ ] Please follow the Ruby name convention to be `posts_counter` instead. Also with the `likes_counter`  and `comments_counter` in the `posts` table

_Kindly apply the changes in all similar situations_

Reply:
@MahmoudBakr23 thank you for your review. Fixed